rfcs/modal-component.md

```diff
@@ -0,0 +1,154 @@
+# Modal Component
+
+### Exports
+
+* `Modal`
+* `ModalHeader`
+* `ModalBody`
+* `ModalFooter`
+* `StyledRoot`
+* `StyledBackdrop`
+* `StyledDialog`
+* `StyledDialogInner`
+* `StyledHeader`
+* `StyledBody`
+* `StyledFooter`
+* `StyledFooterAction`
```

Comment:
I assume this is the container to put those "action" buttons on the right side?
So, these 2 buttons will be rendered side by side on the right side?
```
<StyledFooterAction>
  <button>...</button>
  <button>...</button>
</StyledFooterAction>
```

Reply:
I was thinking StyledFooter would be a flex box with `justify-content: flex-end`, and then StyledFooterAction would wrap _each_ button to create a margin between them.